artifactory/commands/setup/setup.go

```diff
@@ -349,3 +344,26 @@ func (sc *SetupCommand) configureContainer() error {
 		containerManagerType,
 	)
 }
+
+// configureMaven updates the Maven settings.xml file to use the repo Url as mirror.
+func (sc *SetupCommand) configureMaven() error {
+	username := sc.serverDetails.GetUser()
+	password := sc.serverDetails.GetPassword()
+
+	// Get credentials from access-token if exists.
+	if sc.serverDetails.GetAccessToken() != "" {
+		if username == "" {
```

Comment:
Is it possible that the user who created the access token is different from the username in the Username field? If so, I believe it's better to extract the username by default when using an access token.